hw5/task2/script.js

```diff
@@ -0,0 +1,49 @@
+'use strict';
+
+document.addEventListener('DOMContentLoaded', () => {
+    const body = document.body;
+    const mainButton = document.querySelector('.main-btn');
+    const message = document.querySelector('.message');
+
+    if (localStorage.getItem('mainButtonState') === 'on') {
+        applyOnState();
+    } else if (localStorage.getItem('mainButtonState') === 'off') {
+        applyOffState();
+    }
+
+    mainButton.addEventListener('click', () => {
+        if (localStorage.getItem('mainButtonState') === 'off') {
+            localStorage.setItem('mainButtonState', 'on');
+            localStorage.setItem('stateChangeDate', formatDate(new Date()));
```

Comment:
```suggestion
            localStorage.setItem('mainButtonState', 'on');
            localStorage.setItem('stateChangeDate', formatDate(new Date()));
```
Цю логіку також можна винести в applyOnState(); і в applyOffState(); відповідно